daml-script/test/BUILD.bazel

```diff
@@ -174,8 +176,7 @@ da_scala_library(
         "//daml-lf/language",
         "//daml-lf/transaction",
         "//daml-script/runner:script-runner-lib",
-        "//language-support/scala/bindings",
-        "//language-support/scala/bindings-pekko",
+        "//ledger/ledger-api-client",
```

Comment:
This is a sad panda :cry: :panda_face: in its own right. But let's fight that battle another day.

Reply:
Note that the script  and trigger runners are already using the ledger-api-client. 
If you want we drop it, you will have to give us an alternative, we are not really in mood to reimplement a ledger client from scratch.

Reply:
In fact, there is a LedgerClient in the ledger-api-core library of canton. It's a different discussion though whether a dependency on canton's entrails is desirable here.